doc/user_guide/configuration.rst

```diff
@@ -193,7 +193,7 @@ access_mapping          This field defines the strings that are used by a
                         read, write, read+write, increment,
                         read+increment and summation access by
                         PSyclone. Sum is a form of reduction. The
-                        GOcean APIs do not support increment or sum,
+                        GOcean API do not support increment or sum,
```

Comment:
s/do/does/ and s/they only define/it only defines/ below.

Reply:
Done